site/content/docs/main/release-instructions.md

```diff
@@ -3,14 +3,117 @@ title: "Release Instructions"
 layout: docs
 ---
 
-This page covers the steps to perform when releasing a new version of Velero.
-
-## Preparing for a release
+- [General notes](#general-notes)
```

Comment:
ToC can be deleted as it is auto-generated for all pages on the website.

Reply:
Do you need to include `toc: "true"` in the header to generate that?